### PR TITLE
Design Auth Page

### LINKED ISSUE #6: Design authentication page component

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #6 
- This PR contains some other modifications too.

### CODE REVIEW

File: src/components/authenticate/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                        return (
+                            <Chip 
+                                className={classes.chip}
+                                key={i.label}
+                                label={`${i.label.toLowerCase()}${i.required ? ' *' : ''}`} 
+                                variant={i.required ? 'default' : 'outlined'} 
+                            />
+                        )
+                    })
+                }
```
Comment (anthonyminyungi) [inline src/components/authenticate/index.tsx:83]: Appearance of this component.
스크린샷 2020-09-01 오후 12 45 56

File: src/App.tsx [hunk 2]
```diff
@@ -11,28 +11,51 @@ export default function App() {
...
+        label: 'E-mail',
+        required: true,
+      },
+      {
+        label: 'Password',
+        required: true,
+      },
+      {
+        label: 'Birth',
+        required: true,
```
Comment (anthonyminyungi) [inline src/App.tsx:50]: The dummy data for `RequestedInfo` prop type.

File: src/components/common/button/index.tsx [hunk 3]
```diff
@@ -10,6 +10,8 @@ const useStyles = makeStyles(() => ({
         '&:hover': {
             backgroundColor: '#0076f3',
         },
+        fontFamily: ['"Noto Sans"', 'sans-serif'].join(','),
+        fontWeight: 'bold',
```
Comment (anthonyminyungi) [inline src/components/common/button/index.tsx:14]: Font style changed.

**Review by anthonyminyungi — COMMENTED** [review]
Self-review.

**Review by HyeockJinKim — APPROVED** [review]
It looks good.

**Pull request merged**